survey/__init__.py

```diff
@@ -143,7 +143,7 @@ def confirm(*args,
 
     if not 'hint' in kwargs:
         template = helpers.paint('({0}/{1}) ', _colors.hint)
-        kwargs['hint'] = utils.hint.confirm(template, default)
+        kwargs['hint'] = utils.hint.confirm(template, default, color = color)
```

Comment:
The hint's highlight color should not necessarily match the response color. Changing the former should be done by passing a new hint created via `utils.hint.confirm`. Revert this and the PR is good to go!

Reply:
It made sense to me because the hint highlights the default response. But it's your choice of course :)